lib/cog_api/fake/client.ex

```diff
@@ -172,6 +173,24 @@ defmodule CogApi.Fake.Client do
     Rules.delete(rule_id, endpoint)
   end
 
+  def trigger_by_name(%Endpoint{}=endpoint, name),
```

Comment:
What do you think about using the `do/end` here just to be consistent with the rest of the file?